rubrik_cdm/data_management.py

```diff
@@ -1607,6 +1614,14 @@ def create_sla(self, name, hourly_frequency=None, hourly_retention=None, daily_f
                 config["frequencies"]["yearly"]["dayOfYear"] = "LastDay"
                 config["frequencies"]["yearly"]["frequency"] = yearly_frequency
                 config["frequencies"]["yearly"]["retention"] = yearly_retention
+            config["allowedBackupWindows"] = []
```

Comment:
nit: Do we need to create this before checking if `starttime_hour is not None`?

Reply:
You're right, we can move that to just before the append call